src/main/java/com/ericsson/eiffelcommons/helpers/JenkinsXmlData.java

```diff
@@ -113,21 +125,51 @@ public JenkinsXmlData addBashScript(String script) {
      * @return this JenkinsXmlData
      */
     public JenkinsXmlData addGrovyScript(String script) {
-        String hudsonGroovyKey = "hudson.plugins.groovy.Groovy plugin='groovy@2.1'";
-        boolean hasKeyGroovy = builders.has(hudsonGroovyKey);
+        hudsonPluginsGroovyGroovyKey = String.format(hudsonPluginsGroovyGroovyKey, GROOVY_SCRIPT_SCRIPT_PLUGIN_VERSION);
 
+        boolean hasKeyGroovy = builders.has(hudsonPluginsGroovyGroovyKey);
         if (!hasKeyGroovy) {
             JSONArray hudsonTasksGroovy = new JSONArray();
-            builders.put(hudsonGroovyKey, hudsonTasksGroovy);
+            builders.put(hudsonPluginsGroovyGroovyKey, hudsonTasksGroovy);
         }
 
         JSONObject newGroovyCommand = BuildGroovyObject(script);
 
-        builders.getJSONArray(hudsonGroovyKey)
+        builders.getJSONArray(hudsonPluginsGroovyGroovyKey)
                 .put(newGroovyCommand);
         return this;
     }
 
+    /**
+     * This function adds system groovy script to the XML data.
```

Comment:
Write in this description that onyl one groovy script may be added,

Reply:
Done.